src/main/java/com/owncloud/android/lib/resources/status/CapabilityBooleanType.java

```diff
@@ -52,7 +54,7 @@ public static CapabilityBooleanType fromValue(int value)
             case 1:
                 return TRUE;
         }
-        return null;
+        return UNKNOWN;
```

Comment:
wouldn't this be a standard case for switch/case/default?

Reply:
switch refuses to have a default on int. 
maybe I could use <1 and >1, but this is then even stranger…🙈

Reply:
fine for me 👍 Let's keep it this way